django_lightweight_queue/job.py

```diff
@@ -1,21 +1,38 @@
 import sys
 import json
 import time
+import datetime
 
 from .utils import get_path, get_middleware
 
 class Job(object):
-    def __init__(self, path, args, kwargs, timeout=None, sigkill_on_stop=False):
+    def __init__(
+        self,
+        path,
+        args,
+        kwargs,
+        timeout=None,
+        sigkill_on_stop=False,
+        created_time=None,
+    ):
         self.path = path
         self.args = args
         self.kwargs = kwargs
         self.timeout = timeout
         self.sigkill_on_stop = sigkill_on_stop
+        self.created_time = (
+            created_time or datetime.datetime.utcnow().isoformat(sep=' ')
```

Comment:
I don't think we need the `job = cls(**json.loads(val))` property, as we have the `from_json` method. We need to make sure that we preserve the from_json(to_json(x)) fixpoint though, as the reliable redis backend removes job entries from Redis via job.to_json().